manifests/0000_30_07-certified-operators.configmap.yaml

```diff
@@ -1285,7 +1285,7 @@ data:
       
                     containers:
                     - name: mongodb-enterprise-operator
-                      image: quay.io/mongodb/mongodb-enterprise-operator:0.3
+                      image: registry.connect.redhat.com/mongodb/enterprise-operator
```

Comment:
This shouldn't be in your PR. Looks like you may have rebased incorrectly?

Reply:
didn't rebase